packages/devui/src/Tabs/Tabs.tsx

```diff
@@ -24,10 +24,8 @@ export default class Tabs<P> extends Component<TabsProps<P>> {
   SelectedComponent?: React.ComponentType<P>;
   selector?: () => P;
 
-  UNSAFE_componentWillReceiveProps(nextProps: TabsProps<P>) {
-    if (nextProps.selected !== this.props.selected) {
-      this.updateTabs(nextProps);
-    }
+  UNSAFE_componentWillReceiveProps(nextProps: TabsProps<P>): void {
+    this.updateTabs(nextProps);
   }
```

Comment:
If we're going to unconditionally update the tabs can we go ahead and just turn `updateTabs()` into `renderTabs()` and call it from the `render()` method?

Reply:
I think so